settings.gradle.kts

```diff
@@ -63,6 +63,8 @@ project(":kotlinx-serialization-json-tests").projectDir = file("./formats/json-t
 
 include(":kotlinx-serialization-protobuf")
 project(":kotlinx-serialization-protobuf").projectDir = file("./formats/protobuf")
+include(":kotlinx-serialization-protobuf:test-protobuf")
```

Comment:
Given that it is a subproject of `serialization-protobuf`, name `test-protobuf` seems like a repetition. `test-model` or `test-data` seem to fit here more

Reply:
I don't quite agree, because in this case the project name will be `test-model`, what will be perceived ambiguously.
Very often, project names are used (in reports, logs, etc), rather than their full paths. 

But it seems to me that it is very insignificant.

Reply:
How about `test-protobuf-model`?

Reply:
`proto-test-model` then

Reply:
Updated project name